components/rms/slurm/SPECS/slurm.spec

```diff
@@ -16,6 +16,7 @@
 %global _with_slurmrestd 1
 %global _with_multiple_slurmd 1
 %global _with_freeipmi 1
+%global _with_rsmi /opt/rocm/lib
```

Comment:
Why is this a path? This is never used anywhere else.

Reply:
that was leftover from my very first try when I was building manually and I had difference versions of the rocm stack and I wanted to be sure it was using the right one. With a fresh rpm install the path is not required anymore and this can be 1